x-pack/plugin/sql/src/test/java/org/elasticsearch/xpack/sql/planner/QueryTranslatorTests.java

```diff
@@ -1660,6 +1018,8 @@ public void testGroupByHistogramWithDateTruncateIntervalToDayMultiples() {
         }
     }
 
+    // Count
+    ///////////
     public void testCountAndCountDistinctFolding() {
         PhysicalPlan p = optimizeAndPlan("SELECT COUNT(DISTINCT keyword) dkey, COUNT(keyword) key FROM test");
```

Comment:
Is this a leftover? Shouldn't it be externalized?

Reply:
Yep, Done, thx!
I also externalized the ones regarding `trackTotalHits`.